### PR TITLE
also copy rotated logs

### PR DESCRIPTION
**Pull request by boegel**

### CODE REVIEW

**Comment by boegel** [comment]
@stdweird: please review

File: easybuild/tools/filetools.py [hunk 1]
```diff
@@ -875,6 +876,36 @@ def rmtree2(path, n=3):
         _log.info("Path %s successfully removed." % path)
 
 
+def move_logs(from_path, target_path):
```
Comment (stdweird) [inline easybuild/tools/filetools.py]: `from_path` is not really a path but a logfile, maybe call it `main_logfile`?

Comment (stdweird) [inline easybuild/tools/filetools.py]: `target_path` is also no directory?

Comment (boegel) [inline easybuild/tools/filetools.py]: ok, will rename

File: easybuild/tools/filetools.py [hunk 2]
```diff
@@ -875,6 +876,36 @@ def rmtree2(path, n=3):
...
+            # move log to target path
+            shutil.move(app_log, new_log_path)
+            _log.info("Moved log file %s to %s" % (from_path, new_log_path))
+
+    except (IOError, OSError), err:
+        _log.error("Failed to move log file(s) %s* to new log file %s*: %s" % (from_path, target_path, err))
+
+
 def cleanup(logfile, tempdir, testing):
     """Cleanup the specified log file and the tmp directory"""
```
Comment (stdweird) [inline easybuild/tools/filetools.py]: and this method should also support multiple logfiles

**Comment by boegel** [comment]
remarks fixed, Jenkins happy, going in

Thanks for the review @stdweird!

**Pull request merged**